go.mod

```diff
@@ -76,7 +76,7 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.27.0
-	go.viam.com/api v0.1.366
+	go.viam.com/api v0.1.367
```

Comment:
This does not need to be updated. You see things failing due to TabularDataByFilter deprecation warnings

Reply:
Ah good call - bumped back down to v0.1.366